### ISSUE TITLE
Arbitrary HTML in Bootstrap Tab 

### ISSUE THREAD

**Issue by geoffrey-eisenbarth** [django-crispy-forms/django-crispy-forms#1011]
* Package version: 1.8.1
* Django version: 3.0.2
* Python version: 3.7.3
* Template pack: bootstrap3

### Description:

Currently it's not possible to include, e.g., FontAwesome icons in a bootstrap3 tab. I was able to get it working by replacing `{{ link.name|capfirst }}` with `{{ link.name|safe }}` in `templates/bootstrap3/layout/tab-link.html` and then 

```
self.helper.layout = Layout(
  TabHolder(
    Tab(
      '<i class="fa fa-align-left"></i> Align Left',
      Row(<tab_content>),
      css_id='align-left',
    ),
  ),
)
```

I don't know if `|safe` becomes a security issue or if people don't think this functionality is needed, but I like to put icons in my tabs so I figured I'd suggest it to the community. Currently, with the `|capfirst` tag the tab will show the HTML as text instead of rendering the icon. Thoughts?

**Comment by smithdc1** [comment]
Hi @geoffrey-eisenbarth 

I think `|safe` is fine as you are in control of it (presumably you don't want to load malicious html in here yourself 👍 ). `|safe` is used a fair amount in crispy template packs. 

As for incorporating this into the template packs, I'm trying to work out if the change would cause a regression issue? Presumably most people have just a tab name, so may cause some tab names to be lower case...

In conclusion if this is a wide enough use case then I say let's have it. Are you in a position to raise a PR (bs4 pack as well please) and we can get it reviewed more widely? 

Thanks!

**Comment by geoffrey-eisenbarth** [comment]
Good point about the regression issue. Perhaps I'm mistaken, but the examples in [the docs](https://django-crispy-forms.readthedocs.io/en/latest/layouts.html) imply that the first argument in `Tab` should be formatted how you want it to look (e.g., "First Tab"), so I don't think that many users are aware that the text is being run through `|capfirst`. Thoughts?

I don't mind submitting a PR, I should have some time tomorrow in the afternoon.

**Comment by smithdc1** [comment]
Hey, no rush whenever you have time.

Take care.

David
